modules/end-user-guide/pages/developer-workspaces.adoc

```diff
@@ -14,6 +14,21 @@
 * Tool dependencies needed by developers to work on a project
 * Application runtime: a replica of the environment where the application runs in production
 
-Pods manage each component of a {prod-short} workspace. Therefore, everything running in a {prod-short} workspace is running inside containers. This makes a {prod-short} workspace highly portable.
+Pods manage each component of a {prod-short} workspace.
+Therefore, everything running in a {prod-short} workspace is running inside containers.
+This makes a {prod-short} workspace highly portable.
```

Comment:
I don't think splitting the text into separate lines was necessary. It breaks the flow when I read it.

Reply:
It renders as a single paragraph.